package.json

```diff
@@ -32,12 +32,12 @@
   },
   "homepage": "https://github.com/fix/ark-desktop#readme",
   "devDependencies": {
-    "electron-prebuilt": "^1.4.13",
```

Comment:
needs prebuilt to build for different platform (windows, macosx, ubuntu), on a sigle os. TBC